pkg/agent/openflow/pipeline.go

```diff
@@ -234,6 +234,7 @@ const (
 	macRewriteMark = 0b1
 	// cnpDenyMark indicates the packet is denied(Drop/Reject).
 	cnpDenyMark = 0b1
+	bypassHostMark = 0b11
```

Comment:
It's not a special mark `0b11`. `ofPortRegRange` should be set to 3 because it's the ofport of uplink port. Please use `UplinkOFPort` when loading value to reg0.